entries/r/rewind.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Rewind": {
+    "domain": "rewind.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://rewind.com/blog/how-to-set-up-a-2fa-for-rewind/",
+    "keywords": [
+      "backup"
+    ],
+    "regions": [
+        "us",
+        "ca",
+        "uk"
+    ]
```

Comment:
Could you explain why this service is only available in these regions? If they do apply, the code needs to be changed from `uk` to `gb`.

Reply:
@phallobst This was perhaps a mistake by me. We don't restrict service to those areas but I felt those subfolders on the site URLs would be most relevant given all our communications are in English. This could easily be removed as I misinterpreted.

Reply:
People speak English in lots of countries mate :wink:
If that's the only limitation then please remove the regions array.